src/test/scala/com/campudus/tableaux/api/system/SystemAnnotationTest.scala

```diff
@@ -0,0 +1,267 @@
+            "fgColor" -> "#ffffff",
+            "bgColor" -> "#c274ff",
+            "displayName" -> Json.obj(
+              "de" -> "Bitte überprüfen",
+              "en" -> "Please double-check"
+            ),
+            "isMultilang" -> false,
+            "isDashboard" -> true,
+            "isCustom" -> false
+          ),
+          Json.obj(
+            "name" -> "postpone",
+            "priority" -> 3,
+            "fgColor" -> "#ffffff",
+            "bgColor" -> "#999999",
+            "displayName" -> Json.obj(
+              "de" -> "Später",
+              "en" -> "Later"
+            ),
+            "isMultilang" -> false,
+            "isDashboard" -> true,
+            "isCustom" -> false
+          ),
+          Json.obj(
+            "name" -> "needs_translation",
+            "priority" -> 4,
+            "fgColor" -> "#ffffff",
+            "bgColor" -> "#ffae74",
+            "displayName" -> Json.obj(
+              "de" -> "Übersetzung nötig",
+              "en" -> "Translation necessary"
+            ),
+            "isMultilang" -> true,
+            "isDashboard" -> true,
+            "isCustom" -> false
+          )
+        ),
+        annotationConfigs.getJsonArray("annotations")
+      )
+    }
+  }
+
+  // GET /system/annotations/<annotationName>
+
+  @Test
+  def retrieveSingleNonExisting(implicit c: TestContext): Unit =
+    exceptionTest("NOT FOUND") {
+      sendRequest("GET", "/system/annotations/does-not-exist")
+    }
+
+  @Test
+  def retrieveSingle(implicit c: TestContext): Unit = okTest {
+    for {
+      annotationConfig <- sendRequest("GET", "/system/annotations/postpone")
+    } yield {
+      assertJSONEquals(
+        Json.obj(
+          "name" -> "postpone",
+          "priority" -> 3,
+          "fgColor" -> "#ffffff",
+          "bgColor" -> "#999999",
+          "displayName" -> Json.obj(
+            "de" -> "Später",
+            "en" -> "Later"
+          ),
+          "isMultilang" -> false,
+          "isDashboard" -> true,
+          "isCustom" -> false
+        ),
+        annotationConfig
+      )
+    }
+  }
+
+  // DELETE /system/annotations/<annotationName>
+
+  @Test
+  def deleteSingleNonExisting(implicit c: TestContext): Unit =
+    exceptionTest("NOT FOUND") {
+      sendRequest("DELETE", "/system/annotations/does-not-exist")
+    }
+
+  @Test
+  def deleteSingle(implicit c: TestContext): Unit = okTest {
+    for {
+      configsBeforeDeletion <- sendRequest("GET", "/system/annotations").map(_.getJsonArray("annotations"))
+
+      deleteResult <- sendRequest("DELETE", "/system/annotations/postpone")
+
+      configsAfterDeletion <- sendRequest("GET", "/system/annotations").map(_.getJsonArray("annotations"))
+    } yield {
+      assertEquals(4, configsBeforeDeletion.size)
+      assertJSONEquals("""{  "status": "ok" }""", deleteResult.toString)
+      assertEquals(3, configsAfterDeletion.size)
+    }
+  }
+
+  // PATCH /system/annotations/<annotationName>
+
+  @Test
```

Comment:
Bei den patches könnte man evtl. noch abtesten, was mit einem leeren payload object passiert. Ich würde darauf tippen, dass der update query auf Fehler läuft, weil nach dem SET in `Update theTable SET WHERE name = XY` keine column/value pairs kommen.
Falls der payload empty ist, könnte man z.B. eine `InvalidJsonException` werfen.

Reply:
Habe bei Create und Update jeweils noch 2 Tests hinzugefügt:
- leere JSON `{}` -> `error.arguments`
- fehlende payload -> `error.json.notfound`